app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -315,6 +343,9 @@ export default {
     bus.$off(BUS_EVENTS.INSERT_INTO_RICH_EDITOR, this.insertContentIntoEditor);
   },
   methods: {
+    setEditorRoot() {
+      this.editorRoot = this.$refs.editorRoot;
+    },
```

Comment:
You can make this a computed property instead, 

```
editorRoot() {
  return this.$refs.editorRoot;
}
```

Reply:
Done